Source/Extensions/NSURLRequest.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+extension NSURLRequest {
+  var jsonPayload: NSDictionary {
+    let json = HTTPBody.flatMap { try? NSJSONSerialization.JSONObjectWithData($0, options: NSJSONReadingOptions(rawValue: 0)) }
+    return (json as? [String: AnyObject]) ?? [:]
```

Comment:
Just a heads up on this. I ran into some strange issues because I was also using `[:]` as a fallback.

See https://github.com/the-grid/Disc/commit/02b31d935192307749767c8b095aa11f0270e39a and https://github.com/kylef/Mockingjay/issues/32